src/lib/ReleaseHelper.ts

```diff
@@ -3,9 +3,14 @@ import { PropertiesOf } from 'shared/types';
 
 import { IVersion, VersionImpl } from './Version';
 
+export interface IRelease {
+  version: string;
+  active: boolean;
+}
```

Comment:
Switched to a simple dedicated interface, so this could be re-used in both the MAPI and REST API worlds